gamatrix-gog.py

```diff
@@ -331,10 +335,12 @@ def parse_cmdline(argv: List[str]) -> Any:
         sys.exit(1)
 
     cache = Cache(config["cache"])
+    log.info(f"DEBUG: cache state after init: {cache.data.dirty}")
     # Get multiplayer info from IGDB and save it to the cache
     igdb = IGDBHelper(
         config["igdb_client_id"], config["igdb_client_secret"], cache.data
     )
+    log.info(f"DEBUG: cache state after igdb init: {cache.data.dirty}")
 
     if "mode" in config and config["mode"] == "server":
```

Comment:
Would this work? (I know this isn't what this PR is about... sorry 😙)

```suggestion
    if config.get("mode", "no") == "server":
```